file-content-extraction/language-keys/org/apache/tika/language/translate/translator.google.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
these placeholder files are for?

Reply:
This is to leverage tika translation capabilities in the future, can be removed for now

Reply:
dir structure removed @cs-pooja-singh